File: backend/src/main/kotlin/com/letmein/dto/EventDTO.kt
package com.letmein.dto

import org.springframework.web.multipart.MultipartFile
import java.io.File
import java.time.LocalDateTime

data class EventDTO(
    var name: String,
    var startDateTime: LocalDateTime,
    var endDateTime: LocalDateTime,
    var entranceStartTime: LocalDateTime,
    var entranceEndTime: LocalDateTime,
    var registrationEndTime: LocalDateTime,
    var venue: String,
    var description: String,
    var attendeeLimit: Int,
    var image: String? = null,
)

File: frontend/src/app/dto/eventDTO.ts
/**
 * Event Data transfer object for sending via HTTP request
 */
export class EventDTO{
    name: string;
    description: string;
    venue: string;
    startDateTime: Date;
    endDateTime: Date;
    entranceStartTime: Date;
    entranceEndTime: Date;
    registrationEndTime: Date;
    attendeeLimit: number;
    image: File | null;
}
